ejviola/mailroom.py

```diff
@@ -1,104 +1,133 @@
 #!/usr/bin/env python3
 
-# Create a list to hold lists of donations
-donordata = []
+# ______________________________________________________________________________________________________________________
+# IMPORT DEPENDENT MODULES
+import pickle
+import sys
+# ______________________________________________________________________________________________________________________
+# DEFINE GLOBAL VARIABLES
+donor_data_file_name = '/home/vagrant/PycharmProjects/git/Py100-2017q1/ejviola/DonorData.pkl'
+# Open donor data, read donor data into a new list of dicts, close donor data file
+try:
+    donor_data_file = open(donor_data_file_name, 'rb')
+    donor_data = pickle.load(donor_data_file)
+    donor_data_file.close()
+    # Create new donor_list of unique donors based on donor_data
+    donor_list = []
+    for row in donor_data:
+        if row['Name'] not in donor_list:
+            donor_list.append(row['Name'])
+    cause_list = []
+    for row in donor_data:
+        if row['Cause'] not in cause_list:
+            cause_list.append(row['Cause'])
+    report = []
+except:
```

Comment:
Try to never have just a broad `except` catch.  You want to be specific with what exception you want to catch.  In this instance:
`except FileNotFoundError:`
This way if you have another exception like a value error or something, you don't print out the "File could not be opened" text